File: 3242-CountElementsWithMaximumFrequency/3242-CountElementsWithMaximumFrequency.cpp
// Last updated: 01/09/2025, 00:58:20
class Solution {
public:
    int maxFrequencyElements(vector<int>& nums) {
        map<int,int>mp;
        for(auto i : nums){
            mp[i]++;
        }
        int mx = -1;
        for(auto i : mp){
            if(i.second>mx) mx = i.second;
        }
        int cnt = 0;
        for(auto i : mp){
            if(i.second == mx) cnt += i.second; 
        }
        return cnt;
        
    }
};

File: 3242-CountElementsWithMaximumFrequency/3242-CountElementsWithMaximumFrequency.java
// Last updated: 27/12/2025, 21:43:01
class Solution {
    public int maxFrequencyElements(int[] nums) {
        int freq[] = new int[101];
        int maxFreq = 0;
        for(int num : nums){
            freq[num]++;
            maxFreq = Math.max(maxFreq , freq[num]);
        }
        int ans = 0;
        for(int f : freq){
            if(f == maxFreq){
                ans += maxFreq;
            }
        }
        return ans;
    }
}